### PR TITLE
[DOC] Replace WIP PR prefix with PR drafting information

### LINKED ISSUE #221: Add information about draft PRs into contributing guidelines
Original title: Remove WIP information from contributing guidelines
### Original Summary
Now that [GitHub supports draft pull requests](https://github.blog/2019-02-14-introducing-draft-pull-requests/), we can remove the WIP prefix from our contributing guidelines and replace that information with instructions for handling drafts.

### Original Next Steps
1. Remove WIP prefix from contributing guidelines
2. Add information about using GitHub's draft PR feature

### Update
As it turns out, Travis doesn't run on unmergeable PRs (like draft PRs) by design. As a result, linting and unit tests won't be run on draft PRs until they are opened for review _and_ a new, mergeable commit is made. As such, we *do not* want to remove WIP information from the contributing guidelines, and in fact we want to dissuade contributors from using the draft PRs feature, at least for now.

### PR DESCRIPTION
**Pull request by tsalo**
Closes #221.

Changes proposed in this pull request:
- Replace documentation regarding WIP pull request prefix with pull request drafting information in contributing guidelines.

### CODE REVIEW

**Comment by tsalo** [comment]
@emdupre Travis is not being triggered for draft PRs. Any idea how we can deal with this?

**Comment by emdupre** [comment]
I think it's because we have Travis as a GitHub integration. It sounds like the changes to the GitHub API were rolled out after that announcement (or are still being rolled out ?), so we might have to give Travis time to catch up. The cost of being cutting-edge :laughing:

**Comment by tsalo** [comment]
Ah, that makes sense. So I guess I can just change this to "ready for review" then.

**Comment by emdupre** [comment]
Still no ping from Travis. How do you think we should handle this ? I'd prefer to be sure that Travis will ping on draft PRs before converting our workflow.

**Comment by tsalo** [comment]
This isn't a high-priority thing, so I'm fine with waiting.

**Comment by emdupre** [comment]
We'll just have to use other draft PRs to check if we start pinging Travis. I assume any changes here should flag it, since we're back to a "standard" PR now.

**Comment by tsalo** [comment]
I found an actual mistake to fix with a new commit (yay), and Travis is now definitely being triggered.

**Comment by emdupre** [comment]
Sorry, I've been dragging my feet on this because I haven't had a chance to figure out what's happening with Travis. It makes sense that it triggered once your PR was "out of the greys", so to speak, but I'm not sure why it wasn't working when it was a draft PR. I'm going to try to update our travis config.

**Comment by emdupre** [comment]
OK, [in their docs](https://docs.travis-ci.com/user/pull-requests/#my-pull-request-isnt-being-built) it looks like this is intentional:

> If a pull request isn’t built or doesn’t show up in Travis CI’s user interface, that usually means that it can’t be merged. We rely on the merge commit that GitHub transparently creates between the changes in the source branch and the upstream branch the pull request is sent against.

> So when you create or update a pull request, and Travis CI doesn’t create a build for it, make sure the pull request is mergeable. If it isn’t, rebase it against the upstream branch and resolve any merge conflicts. When you push the fixes up to the pull request, Travis CI will happily build them.

Since we rely on Travis for our linting, what do you think we should do ? It looks like it's only triggered if you make a mergeable commit i.e., one _after_ you've updated from a draft to a "real" PR.

**Comment by tsalo** [comment]
Should we abandon the PR draft feature for now? I can update the documentation to say to use the WIP tag and _not_ to use the draft feature, for this very reason. Does that work?

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -135,17 +135,18 @@ When opening a pull request, please use at least one of the following prefixes:
...
 * **[STY]** for stylistic changes
-* **[TST]** for new or updated tests, and
-* **[WIP]** for changes which are not yet ready to be merged
+* **[TST]** for new or updated tests
 
 Pull requests should be submitted early and often!
-If your pull request is not yet ready to be merged, please also include the **[WIP]** prefix.
-This tells the development team that your pull request is a "work-in-progress",
+If your pull request is not yet ready to be merged, please open your pull request as a draft.
+More information about doing this is [available in GitHub's documentation](link_drafts).
```
Comment (KirstieJane) [inline CONTRIBUTING.md]:
```suggestion
More information about doing this is [available in GitHub's documentation][link_drafts].
```

**Comment by KirstieJane** [comment]
LOLZ - disregard my review. I didn't read the whole thread 🤦‍♀️ 

I agree with updating to stick with WIP as suggested in https://github.com/ME-ICA/tedana/pull/222#issuecomment-467123368

**Comment by emdupre** [comment]
If that works for you, @tsalo, I think that makes sense for now (unless Travis changes their minds), and it will make clear to folks why we ask for WIP !

**Comment by tsalo** [comment]
Absolutely. I'll close this PR, open a new one, and update the issue.

**Pull request closed without merging**